filebeat/docs/modules/activemq.asciidoc

```diff
@@ -2,6 +2,8 @@
 This file is generated! See scripts/docs_collector.py
 ////
 
+:edit_url: https://github.com/elastic/beats/edit/main/x-pack/filebeat/module/activemq/_meta/docs.asciidoc
```

Comment:
@gtback One thing I'm undecided about: should we use an attribute to set the correct branch here instead of defaulting to `main`? I can see arguments both ways (especially since older Beats CI is very easy to break). Also, I don't think we have an attribute that resolves to `main`.

Reply:
> Also, I don't think we have an attribute that resolves to main.

You're correct. I don't think we have one. In the backend code I've tried to be more explicit for what is a "version" (like `master`, currently) and what is a "branch" (currently `main`) when those two aren't the same, but I don't think we have attributes for those things. 

I'd probably lean towards just pointing everything to `main` for the time being, until we're ready to deprecate the `master` URLs. I suspect most changes will need to be backported anyway. For issues that are only a problem in older versions, I can see it being confusing (the user clicks an edit link for an old version, but it doesn't match what they're seeing in the `main` branch), but I don't know if how likely that is to be a problem.